### ISSUE TITLE
Alternatives to the vote weighting system

### ISSUE THREAD

**Issue by an0n7mous3** [gridcoin-community/Gridcoin-Tasks#220]
Have we explored in depth some alternatives to the way the voting system is structured? My personal opinion is that this current system needs balancing, so as not to give an excessive degree of influence in polls to any one individual in the community.

As it stands, polls are tallied based on the number of shares. Shares are a reflection of the calculation of one's balance of Gridcoins combined with one's magnitude. This leads to an obvious outcome: The more coins one holds, the more sway that person will have in a poll. This idea has been talked about in the following link.

https://steemit.com/gridcoin/@xaqfields/gridcoin-issue-breakdown-voting-weight

As an alternative to the way one's vote weight is calculated, I propose this idea: We give less weight to the amount of coins one holds, and more weight to one's magnitude. 

It was obvious even in this recent poll that the leading choices were on the upper end of the spectrum, in giving more payout allotted to the research accomplished, versus one's reward for staking. If we are going to view minting in this light, can we not view voting in a similar one as well? 

There are arguments made that those who hold a lot of coins should naturally care more about the coin. But what about those doing BOINC work? Magnitude is a reflection of that, after all. Right now, I could buy 10 million GRC and not have to worry about magnitude, knowing with confidence that any vote I cast will still have more influence over the vast majority of other users. And if we are going to consider alternatives to how votes are weighted, I think we should at least start with looking at that.

Thoughts? I'd love to hear feedback on this. Tell me if I'm wrong, etc.

**Comment by grctest** [comment]
30 days of computation to gain a large rac (maybe even just 14 days?) is
far cheaper than the market cap of gridcoin, I feel that we would be
opening gridcoin polls up to inexpensive manipulation if magnitude was
given more weight than coins.

**Comment by Foggyx420** [comment]
the boinc crunchers already get a boosted weight for voting, where investors are coin for coin.

`MoneySupplyFactor/5.67) * magnitude` + balance for crunchers.

so 10000 coin investor gets 10000 voting weight
and a cruncher with 10000 coins and 50 magnitude at current network conditions would have around 41071 voting weight as of now. thats pretty much 4 times right there.

**Comment by jring-o** [comment]
I agree that coin weight severely outweighs magnitude weight and should be refactored. In the future, I think it could be protocol refactored per block/superblock based on context.

Why is it so simple for those who hold GRC to overpower those who do the crunching when it comes to decision making? This seems like a flawed distribution of power.

If there is a proposal that significantly benefits holders at the cost of crunchers, holders could easily vote it in by themselves. I'm not saying they would, but they have the opportunity. To stress: it's a flaw of the system, not of any people. 

Would it be better for any proposal to have to draw from both types of network participants?

I think a 1:1 ratio of vote-weight would be ideal.

**Comment by RoboticMind** [comment]
I would like to mention these ideas:
https://github.com/gridcoin-community/Gridcoin-Tasks/issues/216
https://github.com/gridcoin-community/Gridcoin-Tasks/issues/219

**Comment by an0n7mous3** [comment]
> 30 days of computation to gain a large rac (maybe even just 14 days?) is
far cheaper than the market cap of gridcoin, I feel that we would be
opening gridcoin polls up to inexpensive manipulation if magnitude was
given more weight than coins.
@grctest - There is also the possibility with our current system that an individual monitors our poll-drafts channel, buys coins in anticipation of the vote, and sells them immediately after using them for it. Theoretically it could cost very little with market value fluctuations to someone who already had the money to begin with.

> and a cruncher with 10000 coins and 50 magnitude at current network conditions would have around 41071 voting weight as of now. thats pretty much 4 times right there.
@Foggyx420 Is this is the metric that's supposed to reflect current network conditions? My own vote does not align with that. I voted with 100,000 GRC @ 58 mag, and my shares according to the block explorer ended up being 138,186. By your explanation, it should have been well over 400,000.

Basically, I came out of the vote feeling like the majority of the influence rests on how much Gridcoin one possesses, and not the amount of BOINC work performed. A 1:1 ratio might be a bit extreme according to some people, but based on my recent experience I think the current ratio aught to be a little closer to it.

@RoboticMind I'd be all for a 'biometrics identity' if it could come with guaranteed security and reliability. It's an interesting concept.

**Comment by barton2526** [comment]
@grc-H202 There were two polls, one for Researchers and one for Investors.  The results of these polls were combined to create the 5.67:1 ratio we use today.  Here are the polls:

https://gridcoinstats.eu/poll/vote_weight_rebalancing_investor_poll
https://gridcoinstats.eu/poll/vote_weight_rebalancing_cruncher_poll

**Comment by grc-H202** [comment]
@barton2526 Interesting. Even the researchers didn't like the 1:1 option. I don't see a whole lot of discussion from back then though? (Just looking at the linked GitHub threads)

**Comment by an0n7mous3** [comment]
I did some digging, and came up with some noteworthy data. According to Gridcoinstats:
- The top 20 CPIDs, ignoring the pools, hold 54,646 (47%) of the total magnitude (115,000). 

Then, I took the total money supply of 405,247,815 GRC, and subtracted 35 million from the foundation wallet and burn address, and have 369.2 million left over. Therefore:
- The top 20 richest CPIDs hold 137,554,000 GRC, which is 37% of coins that can take part in voting. 

I will also note that there is assuredly a decent quantity of coins which have been lost or forgotten about, and will never be able to vote. So 37% is what I'd call a very conservative number, and is most likely higher.

But, even at that, there is only at the very most, a 10% difference in wealth concentration versus magnitude concentration when looking at these numbers. I think this is important to think about when we consider the impacts of balance and magnitude in terms of voting.

**Comment by jring-o** [comment]
@grc-H202 correct there was little discussion and little voter participation.

**Comment by grctest** [comment]
The following proposal (https://github.com/gridcoin-community/Gridcoin-Tasks/issues/268) which is currently being voted on (https://www.gridcoinstats.eu/poll/0bb84592748e2806328d87062054ce9b495392af00c8efa83763f666982b498a) proposes to increase magnitude vote weight by approximately 23.82%, to a total of 41.45% versus balance's 58.54%.

At the time of writing this comment the poll support for this vote weight rebalancing stands at ~94%.

I'd appreciate any comments on the github task issue linked above, or within the reddit thread: https://www.reddit.com/r/gridcoin/comments/1hf8klu/2_new_polls_have_been_created_regarding_proposed/

Hopefully we can close this issue if/when this poll passes? @an0n7mous3

**Comment by grctest** [comment]
With the new vote weight rebalancing authorized and due to be implemented within the next mandatory client upgrade I'm closing this issue.

**Comment by jamescowens** [comment]
I agree.
